scripts/start-ganache

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+BASEPATH="$( dirname "$(dirname "$0")" )"
 mnemonic=$(node -p "require(require('path').join(\"${SCRIPTPATH}\", '../src/ganache-vars')).MNEMONIC")
```

Comment:
This still needs to be changed to `BASEPATH` and to remove the `../` in `../src/..`.